AmericanExpress/build.gradle

```diff
@@ -12,8 +12,6 @@ android {
     defaultConfig {
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
-        versionCode rootProject.versionCode
-        versionName rootProject.versionName
```

Comment:
Android Studio automatically removed these. Apparently they aren't necessary for libraries.